tools/accuracy_checker/accuracy_checker/annotation_converters/antispoofing.py

```diff
@@ -0,0 +1,108 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from .format_converter import DirectoryBasedAnnotationConverter, ConverterReturn
+from ..representation import ClassificationAnnotation
+from ..utils import read_json, check_file_existence
+from ..config import PathField, NumberField
+
+class AntispoofingDatasetConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'antispoofing'
+    annotation_types = (ClassificationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update(
+            {
+                'data_dir': PathField(
+                    is_directory=True, optional=False,
+                    description='path to input images'
+                ),
+                'meta_file': PathField(
```

Comment:
please align parameter name with other converters
```suggestion
                'dataset_meta_file': PathField(
```